ansible_builder/steps.py

```diff
@@ -43,14 +43,19 @@ def __init__(self):
 
 
 class GalaxyInstallSteps(Steps):
-    def __init__(self, requirements_naming):
+    def __init__(self, requirements_naming, keyring):
         """Assumes given requirements file name has been placed in the build context
         """
+
+        install_opts = f"-r {requirements_naming} --collections-path {constants.base_collections_path}"
+        if keyring:
+            install_opts += " --keyring ./keyring.gpg"
+        else:
+            install_opts += " --disable-gpg-verify"
```

Comment:
Looks like we'll have to use the [`ANSIBLE_GALAXY_DISABLE_GPG_VERIFY`](https://docs.ansible.com/ansible-core/devel/reference_appendices/config.html#galaxy-disable-gpg-verify) env var to control this since this option isn't going to be supported on older versions of Ansible.